tools/cli/commands/create.py

```diff
@@ -447,9 +447,53 @@ def create_repo(args, gcloud_repos, repo_name):
     Raises:
       subprocess.CalledProcessError: If the `gcloud` command fails
     """
+    if utils.print_info_messages(args):
+        print('Creating repo {0}'.format(repo_name))
+
     create_cmd = ['create', repo_name]
     utils.call_gcloud_quietly(args, gcloud_repos, create_cmd)
 
+    # Clone the repo into a local scratch directory
+    datalab_dir = tempfile.mkdtemp(prefix='datalab-')
+    clone_cmd = ['clone', repo_name, datalab_dir]
+    utils.call_gcloud_quietly(args, gcloud_repos, clone_cmd)
+
+    # Add a couple of files
+    if utils.print_info_messages(args):
+      print('Adding files to repo using local dir {0}'.format(datalab_dir))
+    with open(datalab_dir + '/.gitignore', 'w') as f:
+      f.write('.ipynb_checkpoints\n')
+      f.write('*.pyc\n')
+
+    with open(datalab_dir + '/README.md', 'w') as f:
+      f.write('# Project Notebooks\n')
+
+    # Commit the new files and push the changes back to the master
+    if utils.print_info_messages(args):
+      print('Commit changes and push repo')
+    try:
+      cmd = 'cd {0} && git add .gitignore README.md'.format(datalab_dir)
```

Comment:
This might be better passing in the directory as a the `cwd` keyword argument to the subprcess invocation.